src/utils/WrapperWrapper.js

```diff
@@ -0,0 +1,48 @@
+const sodiumsumo = require('libsodium-wrappers-sumo');
+
+var rand = async (length) => {
+    await sodiumsumo.ready;
+    return sodiumsumo.randombytes_buf(length);
+}
+
+var salt = async () => {
+    return rand(sodiumsumo.crypto_pwhash_SALTBYTES);
+}
+
+var nonce = async () => {
+    return rand(sodiumsumo.crypto_box_NONCEBYTES);
+}
+
+var keys = async (seed) => {
+    await sodiumsumo.ready;
+    return sodiumsumo.crypto_sign_seed_keypair(seed, '');
+}
+
+var pwhash = async (passphrase, salt) => {
+    await sodiumsumo.ready;
+    return sodiumsumo.crypto_pwhash(sodiumsumo.crypto_box_SEEDBYTES, passphrase, salt, sodiumsumo.crypto_pwhash_OPSLIMIT_INTERACTIVE, sodiumsumo.crypto_pwhash_MEMLIMIT_INTERACTIVE, sodiumsumo.crypto_pwhash_ALG_DEFAULT);
+}
+
+var close = async (message, nonce, key) => {
```

Comment:
These names are extremely generic even though they stand for very precise crypto operations. Is it possible to make them more descriptive?